src/edu/csus/ecs/pc2/ui/team/CreateTeamsTSV.java

```diff
@@ -0,0 +1,268 @@
+                "Purpose: read teams.json file create teams.tsv and groups.tsv files.", //
+                "", //
+                "This will not overwrite output file, use -f to overwrite output file (and "+GROUPS_TSV_FILENAME+")", //
+                "", //
+                "options:", //
+                "--of file  - alternate output filename, default output filename is " + TEAMS_TSV_FILENAME, //
+                "--if file  - alternate input JSON filename, default  filename is " + TEAMS_JSON_FILENAME, //
+                     "-f   overwrite output file, by default will not overwrite output file.", //
+                "", //
+                "On success exit code will be " + SUCCESS_EXIT_CODE, //
+                "Any other exit code is an error.", //
+                "", //
+        };
+
+        for (String s : usage) {
+            System.out.println(s);
+        }
+    }
+
+    protected void writeTSVFile() throws Exception {
+
+        if (!allowOverWrite  && new File(outputFileName).exists()) {
+            fatalError("Output file exists (" + outputFileName + "), will not overwrite.  Use " + OUTPUT_FILE_KEY + " to write to a different file");
+        }
+
+        List<TeamAccount> accounts = loadAccounts(inputFileName);
+
+        String [] header = { "Teams", "1" };
+
+        String headerLine = String.join(TAB, header);
+
+        FileWriter writer = (new FileWriter(new File(outputFileName)));
+
+        writer.write(headerLine + NL);
+
+        for (TeamAccount teamAccount : accounts) {
+            // TODO write teams.tsv output file
+            //            System.out.println(" debug 22 " + teamAccount.toJSON());
+
+            String groupId = "1";
+            if (teamAccount.getGroup_ids().size() > 0) {
+                groupId = teamAccount.getGroup_ids().get(0);
+            }
+            
+            Integer value = groupMap.get(groupId);
+            if (value == null) {
+                value = 0;
+            }
+            groupMap.put(groupId, value.intValue()+1);
+            
+            /**
+                     1  Team Number     22  integer
+2   External ID     24314   integer
+3   Group ID    4   integer
+4   Team name   Hoos    string
+5   Institution name    University of Virginia  string
+6   Institution short name  U Virginia  string
+7   Country Code    USA     string ISO 3166-1 alpha-3 
+             */
+            String [] fields = { //
+                    ""+teamAccount.getId(),
+                    teamAccount.getIcpc_id(), //
+                    groupId, //
+                    teamAccount.getName(), //
+                    teamAccount.getDisplay_name(), //
+                    "",
+                    "XXX", //
+            };
+
+            writer.write(String.join(TAB,fields)+ NL);
+        }
+
+        writer.close();
+        writer = null;
+
+    }
+
+    protected List<TeamAccount> loadAccounts(String jsonFilename) throws JsonParseException, JsonMappingException, IOException {
```

Comment:
Personally, I would prefer to see this method named like `loadAccountsFromJSON`.  Since this class is dealing with both JSON and TSV representations of accounts, I think it would help the reader/developer keep things clearer...